Sources/Error Handling/BackendErrorCode.swift

```diff
@@ -112,6 +117,11 @@ extension BackendErrorCode {
         case .invalidSubscriberAttributes,
              .invalidSubscriberAttributesBody:
             return .invalidSubscriberAttributesError
+        case .couldNotCreateAlias:
+            return .configurationError
```

Comment:
looks weird that this is a configuration error?

Reply:
A bit... The cases where this may happen are:
```
- 'Error creating subscriber alias. Alias limit reached.'
- 'Error creating subscriber alias. Subscriber belongs to another app.'
- 'Error creating subscriber alias. Subscriber already aliased.'
- 'Error creating subscriber alias. Subscriber cannot be aliased to another already aliased subscriber.'
```
I wasn't sure what error code to match it to... considering that these happen if you alias when you shouldn't, I thought `ConfigurationError` was the one that better matched, but open to suggestions... We can create a new error code if you think nothing matches.

Reply:
Yeah I guess that's not too bad. I can't find any other one that would suit better, and I'd rather not create more error codes.